release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -70,10 +70,15 @@ With this release, you can now install {op-system} to iSCSI boot devices. Multip
 
 [id="ocp-4-16-installation-and-update-aws-capi_{context}"]
 ==== Cluster API replaces Terraform for AWS installations
-In {product-title} {product-version}, the installation program uses Cluster API instead of Terraform to provision cluster infrastructure during installations on AWS. There are several additional required permissions as a result of this change. For more information, see xref:../installing/installing_aws/installing-aws-account.adoc#installation-aws-permissions_installing-aws-account[Required AWS permissions for the IAM user].
+In {product-title} {product-version}, the installation program uses Cluster API (CAPI) instead of Terraform to provision cluster infrastructure during installations on AWS. There are several additional required permissions as a result of this change. For more information, see xref:../installing/installing_aws/installing-aws-account.adoc#installation-aws-permissions_installing-aws-account[Required AWS permissions for the IAM user].
 
 Additionally, SSH access to control plane and compute machines is no longer open to the machine network, but is restricted to the security groups associated with the control plane and compute plane machines.
 
+[WARNING]
+====
+Installing a cluster on AWS into a secret or top secret region has not been tested with CAPI as of the release of {product-title} {product-version}. This document will be updated once installation into a secret region has been tested.
```

Comment:
```suggestion
Installing a cluster on AWS into a secret or top-secret region has not been tested with CAPI as of the release of {product-title} {product-version}. This document will be updated after installation into a secret region has been tested.
```
As per [IBM guidelines](https://www.ibm.com/docs/en/ibm-style?topic=word-usage), use the word 'once' only to mean one time. Do not use as a conjunction to mean after or when.